libs/langchain/pyproject.toml

```diff
@@ -129,6 +129,7 @@ markdownify = {version = "^0.11.6", optional = true}
 assemblyai = {version = "^0.17.0", optional = true}
 dashvector = {version = "^1.0.1", optional = true}
 sqlite-vss = {version = "^0.1.2", optional = true}
+elevenlabs = {version = "^0.2.24", optional = true}
```

Comment:
can we skip adding to pyproject? we're trying to move away from having optional dependencies in here and using extra groups, they don't add much to user experience and can cause problems with dev setup

Reply:
shouldn't I add elevenlabs to pyproject to make tests pass?